gensim/models/fasttext.py

```diff
@@ -1,5 +1,27 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+#
+# Copyright (C) 2013 Radim Rehurek <me@radimrehurek.com>
```

Comment:
That doesn't seem right -- and FastText didn't even exist yet in 2013 :)

@manneshiva You can use this block:

```
# Authors: Shiva Manne <s.manne@rare-technologies.com>
# Copyright (C) 2017 RaRe Technologies s.r.o.
```